src/components/application_manager/src/request_info.cc

```diff
@@ -257,6 +257,7 @@ uint32_t RequestInfoSet::RemoveMobileRequests() {
 }
 
 const size_t RequestInfoSet::Size() {
+  sync_primitives::AutoLock lock(this_lock_);
```

Comment:
@LitvinenkoIra 
I saw this comment however from places where it is used you can figure out that it locks *_pending_requests_ containers so I would suggest to rename to something like 'pending_requests_lock_'
To me name 'this_lock_' looks like something temporary or draft as it similar to 'my_lock' , 'tmp' etc. and does not provide any clues for people who read the code.